### ISSUE TITLE
Feature Request: Spotify in Waybar support?

### ISSUE THREAD

**Issue by IgLocoXD** [prasanthrangan/hyprdots#193]
Knowing that you can combine Spotify with the Hyprland theme, It would be nice to have a Spotify track/title in Waybar

**Comment by T-Crypt** [comment]
I'm working on this now and will test today and open a pull request once I check with @prasanthrangan if he wants this integrated into the main dots.

**Comment by T-Crypt** [comment]
@prasanthrangan 

Looks like when I run the wbarconfgen.sh, it removes my , for custom/spotify and messes up my config.jsonc.

Can you take a peak at my dots when you have a chance and see what is causing this? I have to edit the .jsonc in VS and once that is done waybar starts but still having issues with the style and config scripts. I may have overlooked something.

![image](https://github.com/prasanthrangan/hyprdots/assets/132922589/5c58ac07-76a2-404f-b2a0-8744a90b4e70)
![230912_09h41m33s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/132922589/e59d40b8-bf0d-4470-b01a-dd81e716a4f9)

**Comment by T-Crypt** [comment]
@prasanthrangan he just wants the spotify module. I almost have my dots updated and testing

**Comment by prasanthrangan** [comment]
oh cool, thank you so much!

**Comment by T-Crypt** [comment]
I've got this updated in my dots -- Ill send a pull request @prasanthrangan: will you just review to see if I missed anything?

![230912_10h35m39s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/132922589/f1405a00-9072-4898-a9c7-1b6ed2e08b1d)


![230912_10h36m01s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/132922589/528d3107-0d7e-45f8-b38e-5f5f7e668501)

**Comment by T-Crypt** [comment]
@IgLocoXD you can close this issue -- Im submitted my PR to the main branch
